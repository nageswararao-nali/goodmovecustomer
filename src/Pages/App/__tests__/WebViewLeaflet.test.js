import "react-native";
import React from "react";
import { shallow } from "enzyme";
import { WebViewLeaflet } from "../WebViewLeaflet";
import toJson from "enzyme-to-json";
const isValidCoordinates = require("is-valid-coordinates");


describe("<WebViewLeaflet>", () => {
  let mountedComponent;
  let props;
  const sendUpdatedMapCenterCoordsToHTML= jest.fn();

  const mountComponent = () => {
    if (!mountedComponent) {
      mountedComponent = shallow(<WebViewLeaflet {...props} />);
    }
    return mountedComponent;
  };

  beforeEach(() => {
    props = {};
  });

  it("renders correctly", () => {
    expect(mountedComponent).toMatchSnapshot();
  });

  it("accepts valid center coordinates", ()=>{
    props = {
      mapCenterCoords: [38.889931, -77.009003]
    };
    expect(sendUpdatedMapCenterCoordsToHTML).toHaveBeenCalledTimes(1);
  })
});
