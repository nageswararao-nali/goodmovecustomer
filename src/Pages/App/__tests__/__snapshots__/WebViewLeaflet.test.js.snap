// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WebViewLeaflet> renders correctly 1`] = `undefined`;
